pkg/asset/quota/gcp/gcp.go

```diff
@@ -0,0 +1,134 @@
+package gcp
+
+import (
+	"sort"
+
+	"github.com/openshift/installer/pkg/quota"
+	"github.com/openshift/installer/pkg/types"
+)
+
+// Constraints returns a list of quota constraints based on the InstallConfig.
+// These constraints can be used to check if there is enough quota for creating a cluster
+// for the isntall config.
+func Constraints(config *types.InstallConfig) []quota.Constraint {
+	var ret []quota.Constraint
+	for _, gen := range []constraintGenerator{
+		network(config),
+		apiExternal(config),
+		apiInternal(config),
+		others,
+	} {
+		ret = append(ret, gen()...)
+	}
+	return aggregate(ret)
+}
+
+func aggregate(quotas []quota.Constraint) []quota.Constraint {
+	sort.SliceStable(quotas, func(i, j int) bool {
+		return quotas[i].Name < quotas[j].Name
+	})
+
+	i := 0
+	for j := 1; j < len(quotas); j++ {
+		if quotas[i].Name == quotas[j].Name && quotas[i].Region == quotas[j].Region {
+			quotas[i].Count += quotas[j].Count
+		} else {
+			i++
+			if i != j {
+				quotas[i] = quotas[j]
+			}
+		}
+	}
+	return quotas[:i+1]
+}
+
+// constraintGenerator generates a list of constraints.
+type constraintGenerator func() []quota.Constraint
+
+func network(config *types.InstallConfig) func() []quota.Constraint {
+	return func() []quota.Constraint {
+		net := []quota.Constraint{{
+			Name:   "compute.googleapis.com/networks",
+			Region: "global",
+			Count:  1,
+		}, {
+			Name:   "compute.googleapis.com/subnetworks",
+			Region: "global",
+			Count:  500,
```

Comment:
Should this be `2`?

Reply:
https://github.com/openshift/installer/pull/3819/commits/fa77d0e21fe6a36aa59b4497edabe9650c39b368

the actual PR for GCP has the correct number...